html-css/week2/style.css

```diff
@@ -0,0 +1,190 @@
+:root {
+  --white: #ffffff;
+  --lith-gray: #eae8e8;
+  --gray: #808080;
+  --dark-gray: #454444;
+  --black: #000000;
+  --purple: #ab5cdb;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+  color: var(--black);
+}
+
+body {
+  min-width: 20rem;
+}
+
+p {
+  color: var(--gray);
+  margin-bottom: 0.625rem;
+}
+
+h1,
+h2,
+h3 {
+  text-transform: uppercase;
+  margin-bottom: 0.625rem;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  padding: 1.25rem 1.25rem;
+}
+
+.title_section-grey {
+  background-color: var(--lith-gray);
+}
+
+p.article-title {
+  text-transform: lowercase;
+}
+
+.header-nav__items {
+  text-transform: lowercase;
+}
```

Comment:
Since it's the same property and value you could just simply combine all selectors: 
```
p.article-title, .header-nav__items {
  text-transform: lowercase;
}
```